solara/lab/components/menu.py

```diff
@@ -123,13 +142,23 @@ def Page():
     ## Arguments
 
     * activator: Clicking on this element will open the menu. Accepts either a `solara.Element`, or a list of elements.
+    * open: Controls and communicates the state of the menu. If True, the menu is open. If False, the menu is closed.
     * children: List of Elements to be contained in the menu
     * style: CSS style to apply. Applied directly onto the `v-menu` component.
+    * close_on_content_click: If True, clicking on the menu contents will close the menu. If False, the menu will stay open.
```

Comment:
This argument is missing, I see it in #326 but you probably wanted it here.

Reply:
My thought was that `close_on_content_click` was something to allow `InputDate` to work, and therefore shouldn't be here but only in #326. Let me know your thoughts.

Reply:
Ah yes, true, so we should only move the docstring there